src/main/java/com/synopsys/integration/alert/component/settings/SettingsDescriptorActionApi.java

```diff
@@ -66,7 +66,38 @@ public void validateConfig(final Collection<ConfigField> descriptorFields, final
         super.validateConfig(descriptorFields, fieldModel, fieldErrors);
         validateLDAPSettings(fieldModel, fieldErrors);
     }
-    
+
+    @Override
+    public void testConfig(final Collection<ConfigField> descriptorFields, final TestConfigModel testConfig) throws IntegrationException {
+
+    }
+
+    @Override
+    public FieldModel readConfig(final FieldModel fieldModel) {
+        final Optional<UserModel> defaultUser = userAccessor.getUser("sysadmin");
```

Comment:
Should we have a constant somewhere with information about the default user?

Reply:
I'm working on this class right now I can implement these changes.